src/main/resources/application.properties

```diff
@@ -15,3 +15,4 @@ spring.datasource.driver-class-name=org.postgresql.Driver
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.show-sql=false
+spring.session.jdbc.initialize-schema=always
```

Comment:
```suggestion
```
This should be added by the attendees after testing the in-memory approach and before attempting the persisted one

Reply:
This is the one related ot the session. 
This will be removed in further commits along with the spring-session-jdbc dependency.